client/src/web_pages/welcome.jsx

```diff
@@ -2,28 +2,46 @@ import React from 'react';
 import Logo from './logo.jsx';
 
 
-// You may need to import more libraries or files, depending on what's required.
-
-
 export default class Welcome extends React.Component {
     constructor (props) {
         super (props);
         this.state = {
+            SignUp: '',
+            Login: '',
 
         }
+        this.handleChange = this.handleChange.bind(this);
+       
+        
     }
 
+    handleChange(event) {
```

Comment:
Be more specific on this name to avoid conflicts with other functions that may be build/used in the future.